Product Management Service/Code/product_management_service.sql

```diff
@@ -2,16 +2,10 @@ CREATE TABLE products (
   id int PRIMARY KEY,
   name varchar(255),
   price decimal(10,2),
-  category varchar(255) FOREIGN KEY REFERENCES category(product_category)
-);
-
-CREATE TABLE product_details (
-  id int PRIMARY KEY,
+  category varchar(255),
   image blob,
```

Comment:
we cannot have multiple images? Also, is it not a good idea to eep the images separate?